hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/filtermanagement/TargetFilterBeanQuery.java

```diff
@@ -104,8 +105,11 @@ protected ProxyTargetFilter constructBean() {
             final DistributionSet distributionSet = tarFilterQuery.getAutoAssignDistributionSet();
             if (distributionSet != null) {
                 proxyTarFilter.setAutoAssignDistributionSet(new ProxyDistribution(distributionSet));
-                proxyTarFilter.setAutoAssignActionType(tarFilterQuery.getAutoAssignActionType());
+                final ActionType autoAssignActionType = tarFilterQuery.getAutoAssignActionType();
+                proxyTarFilter.setAutoAssignActionType(
```

Comment:
May I ask you to provide a short comment that this is for backward compatibility reasons?
If you only look at the code as of today there will always be a `distributionSet` and an `autoAssignActionType` set.

Reply:
sure